docs/output.md

```diff
@@ -0,0 +1,40 @@
+# Output Definitions
+This page aims to explain and define the various fields that will be present in the output.
+
+## Field Definitions
+
+| Field Name | Description |
+| --- | --- |
+| **Category**| This is the type of risk found by Vault Radar. Eg. secret, PII, or NIL.
+| **Description**| This is a short human readable description or explanation of the risk.
+| **Created At**| This is the time the risk was created or introduced.
+| **Author**| This is the user associated with creating or introducing the risk.
+| **Severity**| This is a classification of the risk. **Critical** risks are things Vault Radar believe are the most deserving of user's immediate attention, followed by **High**, **Medium**, and **Info**.
+| **Is Historic**| This means the risk was first created or introduced in version that precedes the most recent version.
+| **Deep Link**| This is a link the content where the risk was found.
+| **Value Hash**| This is a hash of the secret value itself. This is NOT the value of the secret. Identical hashes, means the secret values are identical.
+| **Fingerprint**| This is a value that is used to distinguish different risk events and incorporates time and location into the value's generation. This is a value that is useful when depending on the output as part of some integration or automation.
+| **Textual Context**| This column may or may not be populated. If it is populated it will be showing text surrounding a found risk. This can be helpful when trying to find a secret in a page or if there multiple secrets on a page.
+| **Activeness**| A positive value here means Vault Radar was able to determine that the secret it found is currently active. We can only check activeness for some of our integrations. Additionally Vault Radar can only identify risks it knows are active. If Vault Radar does not know how to check for activeness or cannot check for activeness, the column will be blank. This is because it is not definitively known in this case whether the secret is actually active or not.
```

Comment:
not fully correct. Radar can detect if a secret is Active, Inactive or activeness is unknown.

Reply:
Oh so we will definitively say it's not active sometimes? Ok so:

Vault Radar will attempt to determine if a secret is active, inactive, or if the activeness is unknown. In the cases where Vault Radar can definitively say a secret is active or inactive the column will populated. In all other cases the column will not be populated to indicate that the status is unknown.

Reply:
Looks good, first statement can be modified to
> Vault Radar will attempt to determine if a secret is active or inactive. In the cases...